app/services/dttp/retrieve_contact.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Dttp
+  class RetrieveContact
+    include ServicePattern
+
+    class HttpError < StandardError; end
+
+    attr_reader :trainee
+
+    CONTACT_ATTRIBUTE_FILTERS = %w[
+      emailaddress1
+      firstname
+      lastname
+      contactid
```

Comment:
```suggestion
```


https://github.com/DFE-Digital/register-trainee-teacher-data/blob/56726004e56996f94c6254e5902dc22553458dfd/app/services/dttp/register_for_trn.rb#L44-L46

https://github.com/DFE-Digital/register-trainee-teacher-data/blob/da0c9760ee1e0c2d410028819d937f0d0e66fb82/app/models/trainee.rb#L141-L143


`contactid` is the `trainee.dttp_id`